front/tsconfig.json

```diff
@@ -9,12 +8,16 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
IMHO we should probably go with `nodenext` so we can keep a broader compatibility and one day maybe have .js.map 😅.

Reply:
done